lib/matplotlib/axes/_base.py

```diff
@@ -421,52 +419,32 @@ def __init__(self, fig, rect,
                  **kwargs
                  ):
         """
-        Build an `~axes.Axes` instance in
-        `~matplotlib.figure.Figure` *fig* with
-        *rect=[left, bottom, width, height]* in
-        `~matplotlib.figure.Figure` coordinates
+        Build an axes in a figure.
 
-        Optional keyword arguments:
+        Parameters
+        ----------
+        fig : `~matplotlib.figure.Figure`
+            The axes is build in the `.Figure` *fig*.
+
+        rect : [left, bottom, width, height]
+            The axes is build in the rectangle *rect*. *rect* is in
+            `.Figure` coordinates.
+
+        sharex, sharey : `~.axes.Axes`, optional
+            The x or y `~.matlotlib.axis` is shared with the x or
```

Comment:
There's a small typo on this line (matlotlib instead of matplotlib). I'm happy to fix this by pushing directly to your branch.